### DISCUSSION TITLE
Sign in on codemagic.io via email is broken for 2 days.

### DISCUSSION THREAD

**Discussion by eli1stark** [codemagic-ci-cd/codemagic-docs#1616] [Q&A] [answered]
Sign in via GitHub works, when you try to use emails auth keys does not arrive.

**Comment by eli1stark** [comment]
I was able to log in but there was some real issue there.

**Comment by mohammedbabelly20** [comment, marked as answer]
Hey @eli1stark
It should be resolved now.
